tests/test_schema_evolution.py

```diff
@@ -1,5 +1,8 @@
 from fastavro import writer as fastavro_writer
-from fastavro.reader import SchemaResolutionError
```

Comment:
I feel like this could be a problem... There are probably projects that import this error: https://github.com/search?p=1&q=SchemaResolutionError&type=Code&utf8=%E2%9C%93.

We might need to add a dummy file or add another way to import this exception and mention it as a breaking change. However, if `reader.py` doesn't need to be named `_reader_py.py` then changing the name back might solve this.

Reply:
I'll see if I can avoid the need to rename. This is similar to the `schema.py` / `_schema.pyx` thing above: mistakes in the library code and/or tests can cause subtle failures and or performance problems.

Reply:
I think I need to keep the renaming, but I created a `reader.py` with a single definition of `SchemaResolutionError` used by both `_reader_py.py` and `_reader.pyx`. Think that will work?